src/module/profile/interface/__test__/profile.spec.ts

```diff
@@ -0,0 +1,94 @@
+import { INestApplication } from '@nestjs/common';
+import { Test } from '@nestjs/testing'
+import { AppModule } from 'src/app.module';
+import * as request from 'supertest'
+import { UpdateProfileDto } from '../../application/dto/update-profile.dto';
+import { TestService } from 'src/module/test/application/service/test.service';
+import { tokens } from './../../../../../src/common/fixtures/user';
+
+describe('Profile', () => {
+    let app: INestApplication;
+    let testService: TestService;
+    beforeAll(async () => {
+        const moduleRef = await Test.createTestingModule({
+            imports: [AppModule]
+        }).compile();
+        app = moduleRef.createNestApplication();
+        await app.init();
+        testService = app.get<TestService>(TestService); 
+        const fixtures = await testService.getEntities()
+        const fixtureWithEntities = testService.entitiesWithFixtures(fixtures)
+        await testService.load(fixtureWithEntities)
+    });
+    
+    describe('GET /profile', () => {
+        it('Should get all profiles', async () => {
+            const response = await request(app.getHttpServer())
+            .get('/profile')
+            expect(response.statusCode).toBe(200);
+            
+            /* const body: Profile[] = response.body;
+            expect(body).toHaveLength(1); */
+        });
+    });
+
+    describe('POST /profile', () => {
+        it('Should create new profile', async () => {
+            const newProfile: any = {
```

Comment:
Avoid using any, create a type based on the actual profile. Like in the FixtureClasses.